test/controllers/actual_db_schema/phantom_migrations_controller_test.rb

```diff
@@ -132,5 +134,12 @@ def active_record_setup
         end
       end
     end
+
+    test "POST #rollback_all changes all phantom migrations status to down and hide migration with down status" do
+      post :rollback_all
+      assert_response :redirect
+      get :index
```

Comment:
is there a way to check list of phantom migrations before and after so that the tests are testing behaviour instead of symptoms only?

Reply:
I'll try